client/login/magic-login/emailed-login-link-successfully.jsx

```diff
@@ -45,14 +45,8 @@ class EmailedLoginLinkSuccessfully extends Component {
 		const { translate, emailAddress } = this.props;
 		const line = [
 			emailAddress
-				? translate( 'We just emailed a link to %(emailAddress)s.', {
-						args: {
-							emailAddress,
-						},
-				  } )
+				? translate( "We've sent a login link to " )
```

Comment:
This "to " looks suspicious.

Reply:
I have removed the space at the end :)
The message should be like this according to 5p3iZ99zPYfl6ronoTYEXO-fi-407%3A2074

<img width="478" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/0cb8eb88-70c8-41e2-878e-c542b23421d8">

Reply:
The address should be part of the translated string, you can use {{email}} or %s. Please don't concat strings, it doesn't work in many languages.

Reply:
I got it. Fixed.